compiler/rustc_passes/src/check_attr.rs

```diff
@@ -373,11 +375,18 @@ impl<'tcx> CheckAttrVisitor<'tcx> {
                             | sym::custom_mir,
                             ..
                         ] => {}
-                        [name, ..] => {
+                        [name, rest@..] => {
                             match BUILTIN_ATTRIBUTE_MAP.get(name) {
                                 // checked below
                                 Some(BuiltinAttribute { type_: AttributeType::CrateLevel, .. }) => {}
                                 Some(_) => {
+                                    if rest.len() > 0 && AttributeParser::<Late>::is_parsed_attribute(slice::from_ref(name)) {
```

Comment:
Currently blocked on https://github.com/rust-lang/rust/pull/143796, is in the merge queue right now should be fixed soon
@rustbot blocked